code/game/objects/items/grenades/syndieminibomb.dm

```diff
@@ -59,11 +59,11 @@
 	update_mob()
 	playsound(loc, 'sound/effects/empulse.ogg', 50, TRUE)
 	radiation_pulse(src, rad_damage)
-	for(var/turf/T in view(freeze_range,loc))
-		if(isfloorturf(T))
-			var/turf/open/floor/F = T
-			F.MakeSlippery(TURF_WET_PERMAFROST, 6 MINUTES)
-			for(var/mob/living/carbon/L in T)
-				L.adjustStaminaLoss(stamina_damage)
-				L.adjust_bodytemperature(-230)
+	for(var/turf/Turf in view(freeze_range,loc))
```

Comment:
This is not a descriptive name.

You need to describe the *context* the variable is in. Re-read https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Also, snake_case.